docs/MockFunctions.md

```diff
@@ -21,46 +21,49 @@ function forEach(items, callback) {
 
 To test this function, we can use a mock function, and inspect the mock's state to ensure the callback is invoked as expected.
 
-```javascript
+```js title="forEach.test.js"
+const forEach = require('./forEach');
+
 const mockCallback = jest.fn(x => 42 + x);
-forEach([0, 1], mockCallback);
 
-// The mock function is called twice
-expect(mockCallback.mock.calls.length).toBe(2);
+test('forEach mock function', () => {
+  forEach([0, 1], mockCallback);
+
+  // The mock function was called twice
+  expect(mockCallback.mock.calls).toHaveLength(2);
 
-// The first argument of the first call to the function was 0
-expect(mockCallback.mock.calls[0][0]).toBe(0);
+  // The first argument of the first call to the function was 0
+  expect(mockCallback.mock.calls[0][0]).toBe(0);
 
-// The first argument of the second call to the function was 1
-expect(mockCallback.mock.calls[1][0]).toBe(1);
+  // The first argument of the second call to the function was 1
+  expect(mockCallback.mock.calls[1][0]).toBe(1);
 
-// The return value of the first call to the function was 42
-expect(mockCallback.mock.results[0].value).toBe(42);
+  // The return value of the first call to the function was 42
+  expect(mockCallback.mock.results[0].value).toBe(42);
+});
 ```
 
 ## `.mock` property
 
 All mock functions have this special `.mock` property, which is where data about how the function has been called and what the function returned is kept. The `.mock` property also tracks the value of `this` for each call, so it is possible to inspect this as well:
 
 ```javascript
-const myMock1 = jest.fn();
-const a = new myMock1();
-console.log(myMock1.mock.instances);
-// > [ <a> ]
+const myMock = jest.fn();
 
-const myMock2 = jest.fn();
+const a = new myMock();
 const b = {};
-const bound = myMock2.bind(b);
+const bound = myMock.bind(b);
 bound();
-console.log(myMock2.mock.contexts);
-// > [ <b> ]
+
+console.log(myMock.mock.instances);
+// > [ <a>, <b> ]
```

Comment:
This looks like accidental change. Could you take another look at Files Changed tab? There are few more files with this change.

Reply:
Doing an edit on the docs I did way too much copy paste, lesson for next time. This was probably introduced because at v28 this block changed but somehow interpreted as older. 
Thank you for the patience, I think we are almost there.